### ISSUE TITLE
sklearn library is deprecated

### ISSUE THREAD

**Issue by tschlenther** [matsim-vsp/matsim-python-tools#37]
`preparation.py` has the following import statement:

`from sklearn.utils import shuffle`

the problem is that sklearn is deprecated. One is supposed to use scikit-learn instead.
